resources/views/pdf/export-template.blade.php

```diff
@@ -155,12 +162,25 @@
                             <td>{{ $status->trainCheckin->origin_stopover->departure_planned?->isoFormat(__('datetime-format')) }}</td>
                             <td>{{ $status->trainCheckin->Destination->name }}</td>
                             <td>{{ $status->trainCheckin->destination_stopover->arrival_planned?->isoFormat(__('datetime-format')) }}</td>
-                            <td>{{ $status->trainCheckin->duration }} min</td>
-                            <td>{{ number($status->trainCheckin->distance / 1000) }} km</td>
-                            <td><i>{{ $status->business->value }}</i></td>
+                            <td class="number-field">{{ $status->trainCheckin->duration }} min</td>
+                            <td class="number-field">{{ number($status->trainCheckin->distance / 1000) }} km</td>
+                            <td class="number-field"><i>{{ $status->business->value }}</i></td>
                         </tr>
                     @endforeach
                 </tbody>
+                <tfoot>
+                    <tr>
+                        <td></td>
+                        <td></td>
+                        <td></td>
+                        <td></td>
+                        <td></td>
+                        <td style="font-style: italic;">Gesamt:</td>
```

Comment:
Oh, I have now seen that the translation is missing here.

Reply:
Fixed with [76e3c06](https://github.com/Traewelling/traewelling/pull/1187/commits/76e3c0693cd1045587641cf7b69002583439ddac).